ansible-ipi-install/roles/installer/tasks/08_cleanup_sub_man_registeration.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: IPI-INSTALL Unregister host via subscription-manager
+  redhat_subscription:
+    activationkey: "{{ activation_key }}"
+    org_id: "{{ org_id }}"
+    state: absent
+  ignore_errors: yes
+
```

Comment:
will every install be using subscription manager?   Should there be a when clause?

Reply:
this should subscribe the box when the credentials variables have been defined